prime-dotnet-webapi/Controllers/OrganizationsController.cs

```diff
@@ -110,6 +123,112 @@ public async Task<ActionResult> CreateOrganization(CreateOrganizationViewModel c
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(typeof(ApiResultResponse<int>), StatusCodes.Status200OK)]
+        public async Task<ActionResult> ClaimOrganization(OrganizationClaimViewModel organizationClaim)
+        {
+            var party = await _partyService.GetPartyAsync(organizationClaim.PartyId, PartyType.SigningAuthority);
+            if (party == null)
+            {
+                return BadRequest("Could not claim an organization, the passed in SigningAuthority does not exist.");
+            }
+
+            if (party.UserId != HttpContext.User.GetPrimeUserId())
+            {
+                return BadRequest("Could not claim an organization, the passed in party does not match current user.");
+            }
+
+            var organization = await _organizationService.GetOrganizationByPecAsync(organizationClaim.PEC);
+            if (organization == null)
+            {
+                return BadRequest("Could not claim an organization, the passed in PEC did not locate an organization.");
+            }
+
+            var orgClaim = await _organizationClaimService.GetOrganizationClaimByOrgIdAsync(organization.Id);
+            if (orgClaim != null)
+            {
+                return BadRequest("Could not claim an organization which has already been claimed.");
+            }
+
+            var organizationId = await _organizationClaimService.CreateOrganizationClaimAsync(organizationClaim, organization);
+
+            return Ok(organizationId);
+        }
+
+        // GET: api/Organizations/claims
+        /// <summary>
+        /// Check if organization claim exists by a given search criteria.
+        /// </summary>
+        [HttpGet("claims", Name = nameof(OrganizationClaimExists))]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(typeof(ApiResultResponse<OrganizationClaim>), StatusCodes.Status200OK)]
+        public async Task<ActionResult> OrganizationClaimExists([FromQuery] OrganizationClaimSearchOptions search)
+        {
+            var result = await _organizationClaimService.OrganizationClaimExistsAsync(search);
+
+            return Ok(result);
+        }
+
+        // GET: api/Organizations/5/claims
+        /// <summary>
+        /// Find OrganizationClaim by Organization ID.
+        /// </summary>
+        [HttpGet("{organizationId}/claims", Name = nameof(GetOrganizationClaimByOrgId))]
+        [Authorize(Roles = Roles.ViewSite)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(typeof(ApiResultResponse<OrganizationClaim>), StatusCodes.Status200OK)]
+        public async Task<ActionResult> GetOrganizationClaimByOrgId(int organizationId)
```

Comment:
As alluded to in the previous comment, we don't need this endpoint.

Reply:
@johnnyl6 @neophyte57 has this been addressed?  Trying to resolve remaining issues so we can get this PR out the door.

Reply:
As per the other conversation, we still need this endpoint:

![image](https://user-images.githubusercontent.com/7586665/127240377-fcd78c9a-b5aa-460d-b1c3-4ba82ea5bc45.png)